### ISSUE TITLE
When multiple configuration files are set for the detekt-intellij-plugin,  only the rules enabled in all configuration files are used to analyze the code.

### ISSUE THREAD

**Issue by Lovelemon88** [detekt/detekt-intellij-plugin#543]
<img width="1195" height="784" alt="Image" src="https://github.com/user-attachments/assets/1c683752-97c6-4782-8860-28662cf5a9ab" />
for example ： 

in moduleA: configA.yml

 ```
ruleset1：
	active： true
	rule1：
		active： true
	rule2：
		active： true
 ```

in moduleB: configB.yml
 
```
ruleset1：
	active： true
	rule1：
		active： true
	rule2：
		active： false		
 ```
Both  moduleA and moudleB are belong to the project.
Ultimately, the IDE only displays code that violates rule1 within ruleset1, and code that violates rule2 is not shown when multiple configuration files are set. However, if only configA.yml( in moduleA subproject)is configured, violations of both rule1 and rule2 are detected in the code.
